src/main/kotlin/model/NationalPrisonRadio.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.hmpps.architecture
+
+import com.structurizr.model.Location
+import com.structurizr.model.Model
+import com.structurizr.model.SoftwareSystem
+import com.structurizr.view.ViewSet
+
+class NationalPrisonRadio private constructor() {
+  companion object : HMPPSSoftwareSystem {
+    lateinit var system: SoftwareSystem
+
+    override fun defineModelEntities(model: Model) {
+      system = model.addSoftwareSystem(
+        "National Prison Radio",
+        "The national radio station for prisoners. Made by prisoners, for prisoners"
+      ).apply {
+        Tags.PROVIDER.addTo(this)
```

Comment:
🤔 Double-checking we mean the same thing: the other usages of `PROVIDER` mean an HMPPS-contracted third party providing a service. Is it the same here? Should we change the name (later) to reflect this?

Reply:
:+1: to making the name more meaningful in future, as I agree there's a bit of ambiguity. Fortunately here NPR is a third-party service HMPPS have contracted to provide prison radio. The livestream is the only thing we reach out to at the moment, but their team upload radio shows... and now I've realised there's another user to add to the diagram 🤪